src/Service/Nph/NphDlwBackfillService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Service\Nph;
+
+use App\Entity\NphDlw;
+use App\Repository\NphDlwRepository;
+use Doctrine\ORM\EntityManagerInterface;
+
+class NphDlwBackfillService
+{
+    protected EntityManagerInterface $em;
+    protected NphDlwRepository $NphDlwRepository;
+    protected NphOrderService $nphOrderService;
+
+    public function __construct(EntityManagerInterface $em, NphDlwRepository $dlwRepository, NphOrderService $nphOrderService)
+    {
+        $this->em = $em;
+        $this->NphDlwRepository = $dlwRepository;
```

Comment:
@Ehaic I think you can remove this as you are directly calling the repository using $this->em->getRepository on line 24